django_tables2/utils.py

```diff
@@ -303,7 +303,12 @@ class Accessor(str):
         "Failed lookup for key [{key}] in {context}, when resolving the accessor {accessor}"
     )
 
-    def __new__(cls, value):
+    def __init__(self, object, *callable_args, **callable_kwargs):
+        self.callable_args = callable_args
+        self.callable_kwargs = callable_kwargs
+        super().__init__()
+
+    def __new__(cls, value, *args, **kwargs):
```

Comment:
`__new__` did not have `*args, **kwargs` before, does the fix need that?

Reply:
__new__ and __init__ need to have compatible signatures,  I updated it so that it is explicit now